server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/job/PartitionPlanJob.java

```diff
@@ -129,6 +141,16 @@ public void execute(JobExecutionContext context) {
                     paramemters.getTimeoutMillis(), paramemters.getErrorStrategy());
         } catch (Exception e) {
             log.warn("Failed to execute a partition plan task", e);
+            AlarmUtils.alarm(AlarmEventNames.PARTITION_PLAN_SCHEDULED_FAILED, e);
```

Comment:
not stable, we cannot garantee no exception thrown here, we may enforce logic in AlarmUtils.alarm